others/RESOURCES.md

```diff
@@ -11,3 +11,10 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Interesting documentaries about different technologies
```

Comment:
Is it only about technologies?

Reply:
Yes. The documentary about React I sent is from this channel.

Reply:
Same here as the comment above. Furthermore, just by clicking on the link I see a playlist of mini-docs which are not focused on technologies but people and how they deal with certain situations. 

In most cases, tech channels often cover a variety of topics. The main goal of this list of resources is to provide a quick insight into what they can offer to the user.

Reply:
Done